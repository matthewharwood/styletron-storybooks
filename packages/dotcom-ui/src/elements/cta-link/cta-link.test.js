import {CtaLink} from './cta-link';

test('render a CtaLink', () => {
  const wrapper = shallow(
    <CtaLink
      aria-label={'hello'}
      href={'http://www.google.com'}
      $style={'button'}
      $size={'lead'}>
        Hello Jest!
    </CtaLink>
  );
  expect(wrapper).toMatchSnapshot();
});