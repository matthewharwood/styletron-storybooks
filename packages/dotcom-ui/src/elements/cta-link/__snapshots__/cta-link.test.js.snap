// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render a CtaLink 1`] = `
<Styled(a)
  $size="lead"
  $style="button"
  aria-label=""
  href="http://www.google.com"
>
  1234 Test
</Styled(a)>
`;
